CHANGELOG.md

```diff
@@ -3,6 +3,71 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## v2.0.0 - [2024-07]
```

Comment:
Breaking updates in the sense of "requiring a major version update" was just the change of some parameter names (in #391), but more functional impact have the addition of taxonomy workflow (gray line in metro map), adding pre-annotated input (see metro map), adding argNorm (see metro map), and adding nf-test. So maybe the metro map emphasizes those enough already – or shall we have an extra section "breaking changes" for major version updates, @jfy133 @Darcy220606?

Reply:
Breaking changes is the chaning of parameter names.

I would simply highlight with `BREAKING CHANGE:` before the description

new functionality such as taxonomy workflow or argnorm doesn't break an existing interaction with the pipeline - but someones automated script that sassums parameter names may break (For example)

Reply:
I did it inline now. Tried also how it looks as a separate section, but find it a bit misleading. "Breaking" is strictly the change of some parameters, but user might expect the biggest functional updates mentioned above? So keeping it inline but visible seems the best compromise for me.

![image](https://github.com/user-attachments/assets/8125be94-be59-4f97-88a0-f0d15f1b1a79)

Reply:
instead of pointing to an PR with not too descriptive name I would specify what parameter changed (if you don't want to do that in the changelog, you can do it in the release notes)

Reply:
Good idea actually. I updated the changelog to include a table of changed parameters (see [commit](https://github.com/nf-core/funcscan/pull/410/commits/04195adca3f24cc22b15ed23385a9d63ecb25ceb)).